libraries/standard/mqtt/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# Include filepaths for source and include.
```

Comment:
I suggest the same here of merging `mqtt/CMakeLists.txt` and `mqtt/filepaths.cmake` into a single `mqtt/CMakeLists.txt`

Reply:
Oh the reason why it's separated here is to make it easier to port to AFR as discussed by Michael and Lee.